src/commands/move.json

```
@@ -52,7 +52,7 @@
                     "const": 1
                 },
                 {
-                    "description": "Key wasn't moved.",
+                    "description": "When key already exists in the destination database, or it does not exist in the source database",
```

Lets also keep the old wording? The short one is a good wording too i think
```suggestion
                    "description": "Key wasn't moved. When key already exists in the destination database, or it does not exist in the source database",
```